src/vivarium/framework/population.py

```diff
@@ -220,15 +222,22 @@ def _order_initializers(self) -> None:
             starting_length = len(unordered_initializers)
             for _ in range(len(unordered_initializers)):
                 initializer, columns_created, columns_required = unordered_initializers.pop()
+                initializer_created.extend(columns_created)
+                initializer_required.extend(columns_required)
                 if set(columns_required) <= set(available_columns):
```

Comment:
@collijk  Actually, I don't see how this was ever true, because required contains 'tracked' and, to the best of my knowledge, creates never does? It must be that tracked gets created ?

Reply:
NVM